CoreMapView/src/au/gov/asd/tac/constellation/views/mapview/providers/EsriMapProvider.java

```diff
@@ -21,20 +21,23 @@
 
 /**
  * A tiled base map being served by an ArcGIS Server. This class will ensure
- * tiles are exported in the Web Mercator projection with the correct alignment.
+ * tiles are exported in the Web Mercator projection with the correct alignment
+ * by exporting tiles as images.
  *
  * @author cygnus_x-1
  */
-public abstract class ArcgisMapProvider extends MapProvider {
+public abstract class EsriMapProvider extends MapProvider {
```

Comment:
Please document this breaking change in `CHANGELOG.md`

Reply:
Done.